plugins/woocommerce/package.json

```diff
@@ -474,22 +475,13 @@
 					"command": "test:perf",
 					"optional": true,
 					"changes": [
-						"client/admin/config/*.json",
-						"composer.json",
-						"composer.lock",
-						"includes/**/*.php",
-						"patterns/**/*.php",
-						"src/**/*.php",
-						"templates/**/*.php",
-						"tests/performance/**",
-						".wp-env.json",
-						"woocommerce.php",
-						"uninstall.php"
+						"tests/performance/**"
 					],
 					"testEnv": {
-						"start": "env:perf"
+						"start": "test:perf:ci-setup"
 					},
 					"events": [
+						"pull_request",
```

Comment:
We should leave `pull_request` but only with `"tests/performance/**"` in the changes array. This way the K6 test will always run when they are updated.